libqtile/backend/x11/core.py

```diff
@@ -211,7 +211,10 @@ def wmname(self, wmname):
         self._wmname = wmname
         self._supporting_wm_check_window.set_property("_NET_WM_NAME", wmname)
 
-    def setup_listener(self, qtile: "Qtile") -> None:
+    def set_qtile(self, qtile: "Qtile") -> None:
+        self.qtile
```

Comment:
This doesn't look right.

Reply:
Yep, thanks. leftover from when I was considering using a setter; I deleted it.